<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWidget</class>
 <widget class="QWidget" name="ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="configSel"/>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="configStack">
     <widget class="QWidget" name="NoCWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>No configuration loaded</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="PWConfig" name="PWWidget"/>
     <widget class="LmpConfig" name="LmpWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PWConfig</class>
   <extends>QWidget</extends>
   <header>qt/configs/pwconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LmpConfig</class>
   <extends>QWidget</extends>
   <header>qt/configs/lmpconfig.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
